[Optimize] add issue template

gaoachao opened the pull request:
Add issue template
- Bug Report
- Feature Request

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/lynx-family/lynx?pullRequest=42) <br/>All committers have signed the CLA.

gaoachao committed:
[Optimize] add issue template

Add standardized issue templates:
- bug-report.md: For structured bug reporting
- feature-request.md: For organized feature requests

These templates will help contributors provide more complete information,
streamline issue triage, and improve overall project management efficiency.

jianliang00 commented:
/land

jianliang00 commented:
🛬 Landing process started! >>208.0
